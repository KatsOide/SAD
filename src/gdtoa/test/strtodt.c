/Users/oide/SAD/oldsad/contrib/gdtoa/test/strtodt.c