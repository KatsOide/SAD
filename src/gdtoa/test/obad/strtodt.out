/Users/oide/SAD/oldsad/contrib/gdtoa/test/obad/strtodt.out